src/cards/arrow3.ts

```diff
@@ -5,14 +5,14 @@ import { arrowEffect } from './arrow';
 import { arrow2CardId } from './arrow2';
 
 export const arrow3CardId = 'Arrow3';
-const damageDone = 30;
+const damageDone = 40;
```

Comment:
This is a pretty significant jump. Why this change?

Reply:
Slash goes 20-40-80, this makes arrow go 10-20-40.

Also, Arrows are very bad due to having low damage. You can't reliably use it as a primary damage spell because you will have to cast it twice as much as slash, and this means the cost will scale significantly, and wont come back for turns at a time. Archer solves this problem, but I'd like the spell to be usable for all classes

Reply:
Hmm, well arrows do have the advantage of infinite range.  We'll have to play test it a bit and see.  I see your logic here